app/Model/CoGroup.php

```diff
@@ -377,6 +429,14 @@ public function addDefaults($coId, $couId=null, $rename=false) {
         'status'      => SuspendableStatusEnum::Active,
         'cou_id'      => ($couId ?: null)
       ),
+      ':approvers' => array(
```

Comment:
Minor nit: I would keep this list alphabetical, and move `approvers` after `admins`.